docker-entrypoint.sh

```diff
@@ -62,11 +62,9 @@ mkfs.xfs $ROOTFS
 #
 
 ETCD_DATA_VOLUME_PATH=""
-SECRETS_DATA_VOLUME_PATH=""
```

Comment:
We do not need the secrets anymore since the service account key-pair is created by the cert-operator, which is injected via the cloud-config.